.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -380,6 +380,13 @@ stages:
                     echo "*** Load credentials for AKS Cluster $aksClusterName in $aksClusterResourceGroup"
                     az aks get-credentials --name $aksClusterName --resource-group $aksClusterResourceGroup --overwrite-existing
 
+                    $ingressNamespace = "ingress-nginx"
```

Comment:
Looks good to me. Just one minor thing. We've specified the workload and the chaos namespace in our `configuration.yaml`. Would it make sense to move the ingressNamespace to this file as well?

Reply:
you didnt complain about this in the original PR :D but of course you are right! will change

Reply:
Sorry.